### ISSUE TITLE
IO user creates new Operator

### ISSUE THREAD

**Issue by andrea-williams** [bcgov/cas-registration#103]
#### Description:

As an IO, I want to be able to create a new Operator in the Registration app so that I can become prime admin for the Operator.

#### Acceptance Criteria:

Given I have logged in to my Business BCeID account
When my company does not appear in the list of Operators
Then I can submit a new Operator that will be saved to the Registration database
And submitting the Operator is also a request to create an Operator profile and make myself a prime admin

#### Dependencies:
- Blocked by #24
- Blocking #299 

#### Development Checklist:

- [x] what data field(s) will be used to ensure there aren't duplicate Operators created? <- CRA Business Number
- [x] on submission of the form,  the record for the Operator is automatically saved to the database ~(no validation/review from CAS internal staff required)~. A request to register is also submitted (this does require review from CAS)
- [x] see wireframes: https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=2615-3283&mode=design&t=rbpZ5ZBSvbVYvBzB-4
- [x] - [ ] move the checkbox asking whether the user requesting prime admin is the Senior Officer to be the first field in the form
- [x] if the user requesting prime admin IS the Senior Officer, we should be able to pre-populate some of the Senior Officer contact fields with fields we have about the user (from the `user` table, such as name and email)
- [x] else if the user requesting prime admin IS NOT the Senior Officer, the user should be prompted to input contact information only about the Senior Officer, not about themselves. (We may also display a list of existing contacts associated with the Operator, but that will be a subsequent ticket)
- [x] store contact information about the operator's Senior Officer in the `contact` table, not the user table
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [x] User story is included
- [ ] User role and type are identified
- [x] Acceptance criteria are included
- [x] Wireframes are included (if required)
- [x] Design / Solution is accepted by Product Owner
- [x] Dependencies are identified (technical, business, regulatory/policy)
- [x] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [x] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [x] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [x] Ticket is ready to be merged to main branch
- [x] Can be demoed in Sprint Review
- [x] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

-

**Comment by Dianadec** [comment]
Update this card after today's dev/design sync

**Comment by BCerki** [comment]
I think is blocked on 24 because 24 involves creating the form to collect Operator info

**Comment by hannavovk** [comment]
I think this is actually covered by #26. Or are there additional aspects here that are yet to be worked on?  @andrea-williams, @BCerki - can you confirm?

**Comment by hannavovk** [comment]
Update: #26 is for operation, this is for operator.

**Comment by andrea-williams** [comment]
Updated Development Checklist: we've received revised direction from PO that when a new Operator is created it will need to be reviewed by CAS internal staff
